admin-dev/themes/new-theme/scss/components/layout/_nav_bar.scss

```diff
@@ -476,22 +486,18 @@
 
 .category-title {
   display: block;
-  padding: 0.875rem 0 0.75rem;
-  margin: 1.813rem 0 0 0.938rem;
+  padding: 0.5rem 0 0.375rem;
+  margin: 0.875rem 0 0 0.938rem;
   font-size: 0.75rem;
   font-weight: 700;
-  border-top: 1px solid #bbcdd2;
 
   @include media-breakpoint-down(md) {
-    margin: 0.75rem 0 0 0.5rem;
+    margin: 0.75rem 0 0 1rem;
     font-size: 1rem;
-    font-weight: 500;
-    border-top: none;
   }
 
   & > .title {
-    color: $white;
-    background: $gray-dark;
+    color: #6c868e;
```

Comment:
Oh wow sorry forgot this one and I've another problem

Reply:
It should be $gray-medium
I'm testing...